### PR TITLE
Added example in docstring

### LINKED ISSUE #120: Add an example to MLMMaskGenerator's docstring to show the best practice
[MLMMaskGenerator](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/layers/preprocessing/mlm_mask_generator.py) is a helpful tool to generate masks for input sequences. The current docstring only has two basic examples, and it would be better for users if we can provide an example of how it should be used in a real MLM workflow. 

Basically the example will have:
1. Prepare data, we can generate random strings or load some small datasets from TFDS (tensorflow datasets).
2. Instantiate a tokenizer, and call tokenize on the data.
3. Apply `MLMMaskGenerator` on the tokenized data.
4. Feed the masked data to a dummy MLM model by calling `model.fit()`.

### PR DESCRIPTION
**Pull request by prajakta-1527**
closes #120

### CODE REVIEW

**Comment by mattdangerw** [comment]
I think this is fixed already. Thanks for the contribution!
